save-common/src/commonMain/kotlin/org/cqfn/save/core/utils/PlatformUtils.kt

```diff
@@ -13,6 +13,23 @@ enum class CurrentOs {
     LINUX, MACOS, UNDEFINED, WINDOWS
 }
 
+/**
+ * Atomic values
+ */
+@Suppress("VARIABLE_NAME_INCORRECT_FORMAT", "ConstructorParameterNaming")
```

Comment:
Since we are not using typealias for native anymore, we can use proper variable naming. Also, what does diktat complain about here?